packages/manager/apps/dedicated/client/app/dedicated/server/installation/ovh/dedicated-server-installation-ovh.controller.js

```diff
@@ -958,7 +960,31 @@ angular
 
       // ------Add partition------
 
+      function isMountPointAlreadyUsed(newMountPoint) {
+        return $scope.installation.partitionSchemeModels.some(
+          (partition) => partition.mountPoint === newMountPoint,
+        );
+      }
+
+      function getRandomMountPoint() {
+        const alphabet = 'defghijklmnopqrstuvwxyz'; // mountpoint character will be within c and z alphabet and there can be maximum 24 partitions
```

Comment:
```suggestion
        const alphabet = 'defghijklmnopqrstuvwxyza'; // mountpoint character will be within c and z alphabet and there can be maximum 24 partitions
```
So the 25th partition that raises the error from APIv6 will be with "a:" mountpoint rather than ":"

Reply:
![image](https://user-images.githubusercontent.com/17429937/219600888-1445ed26-22a8-4db6-8f2a-60c2c36be08f.png)
This suggestion is so that it doesn't show this confusing 2nd error

Reply:
Okay will add it to avoid confusion in Error message.